Sources/AppcuesKit/Presentation/Traits/Appcues/DialogContainerView.swift

```diff
@@ -43,7 +44,7 @@ internal class DialogContainerView: UIView {
             dialogView.bottomAnchor.constraint(lessThanOrEqualTo: safeAreaLayoutGuide.bottomAnchor),
             dialogView.centerYAnchor.constraint(equalTo: centerYAnchor),
             // this is required so the dialogView has an initial non-zero height, after which it can start sizing to the content.
-            dialogView.heightAnchor.constraint(greaterThanOrEqualToConstant: 1),
+            dialogView.layoutMarginsGuide.bottomAnchor.constraint(greaterThanOrEqualTo: dialogView.layoutMarginsGuide.topAnchor, constant: 1),
```

Comment:
@iujames this line fixes the 0 height issue with carousels. What was happening is if the layout margins from the border took the 1px of space, then the `dialogView` would never get laid out. I've rewritten the constraint to ensure there's always the necessary 1px regardless of the layout margins.

Reply:
👍 Good find - tested that modal out locally here and it looks good now.